packages/xo-web/src/xo-app/hub/resource-form.js

```diff
@@ -52,9 +79,52 @@ export default decorate([
           onChange={effects.onChangePool}
           predicate={poolPredicate}
           required
-          value={state.pools}
+          value={pools}
         />
       </FormGrid.Row>
+      {install && isMultiplePools ? (
+        <div>
+          <SingleLineRow>
+            <Col size={6}>
+              <strong>{_('pool')}</strong>
+            </Col>
+            <Col size={6}>
+              <strong>{_('vdiSr')}</strong>
+            </Col>
+          </SingleLineRow>
+          <hr />
+          {pools.map(pool => (
+            <SingleLineRow key={pool.uuid} className='mt-1'>
+              <Col size={6}>{pool.name_label}</Col>
+              <Col size={6}>
+                <SelectSr
+                  onChange={effects.onChangeSr}
+                  predicate={sr => sr.$pool === pool.id && isSrWritable(sr)}
+                  required
+                  value={mapPoolsSrs[pool.id]}
+                />
+              </Col>
+            </SingleLineRow>
+          ))}
+        </div>
+      ) : (
+        install && (
+          <FormGrid.Row>
+            <label>{_('vmImportToSr')}</label>
+            <SelectSr
+              disabled={isSrSelectDisabled}
+              onChange={effects.onChangeSr}
+              predicate={sr =>
+                find(pools, { id: sr.$pool }) !== undefined && isSrWritable(sr)
```

Comment:
If you want a `boolean` as result better use `some`.  `pools` isn't an array in this case.

Reply:
@pdonias think that `find` is more clear... let's wait for the review of reviews to finish